dashboard/db/migrate/20180305045530_add_topic_ethical_social_to_census_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddTopicEthicalSocialToCensusSubmissions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :census_submissions, :topic_ethical_social, :boolean
```

Comment:
Please add a `comment:` description. It's not obvious to me what topic_ethical_social represents.